alerts/jira.go

```diff
@@ -22,7 +22,7 @@ import (
 
 // DescriptionTemplateString defines the content of the alert ticket.
 const DescriptionTemplateString = `
-h2. [{{.Task.DisplayName}} failed on {{.Build.DisplayName}}|{{.UIRoot}}/task/{{.Task.Id}}/{{.Task.Execution}}]
+h2. [{{.Task.DisplayName}} failed on {{.Build.DisplayName}}|{{.UIRoot}}/task/{{.Task.Id | urlquery}}/{{.Task.Execution}}]
```

Comment:
There's no template function called urlquery

Reply:
Supposedly there is according to https://golang.org/pkg/text/template/. Seems it was introduced here: https://github.com/golang/go/commit/5e48e648f1729d02c0a9c411759f8eff9b5d9cb3.

Reply:
I think it's actually urlescape that isn't a thing. I apparently changed it in jira_test (because it had an obvious test avenue to expose it), but spaced on the other places.

Reply:
It seems like it should be "urlquery" not "urlescape"